graphql_compiler/compiler/emit_match.py

```diff
@@ -4,9 +4,11 @@
 
 import six
 
+from graphql_compiler.helpers import get_only_element_from_collection
```

Comment:
When writing a python package, please always use relative imports (`from .helpers`) instead of absolute ones (`from graphql_compiler.helpers`) to reference the package itself. You can use `..` to go up a directory.